torchaudio/functional/functional.py

```diff
@@ -464,6 +484,8 @@ def melscale_fbanks(
 ) -> Tensor:
     r"""Create a frequency bin conversion matrix.
 
+    .. devices:: CPU
+
```

Comment:
`melscale_fbanks`, `linear_fbanks`, and `create_dct` are all torchscript-tested as well